packages/forms/src/Components/BaseFileUpload.php

```diff
@@ -231,6 +233,15 @@ public function directory(string | Closure | null $directory): static
         return $this;
     }
 
+    public function s3(string | Closure $diskName = 's3', bool | Closure $presigned = false): static
```

Comment:
will it throw an error is this api is used on a not supported driver?

Reply:
Yes, Filesystem handles that already

Reply:
that would happen after an upload is attempted or when the form is rendered?

Reply:
Upload attempted